spec/mem/pointer.md

```diff
@@ -65,22 +65,39 @@ impl<Provenance> ThinPointer<Provenance> {
         }
     }
 }
+
+impl PointerMetaKind {
+    pub fn matches(self, meta: Option<PointerMeta>) -> bool {
+        match (self, meta) {
+            (PointerMetaKind::None, None) => true,
+            _ => false,
+        }
+    }
+}
 ```
 
 ## Pointee
 
 We sometimes need information what it is that a pointer points to, this is captured in a "pointer type".
+However, for unsized types the layout might depend on the pointer metadata, which gives rise to the "size strategy".
 
 ```rust
 /// Describes what we know about data behind a pointer.
 pub struct PointeeInfo {
     pub meta_kind: PointerMetaKind,
```

Comment:
This field should not be needed any more, it is defined by size strategy, right?